.github/workflows/deploy.yml

```diff
@@ -113,10 +112,10 @@ jobs:
           $SSH "#! /bin/bash -e
             $TUTOR config save --append 'OPENEDX_EXTRA_PIP_REQUIREMENTS=edx-event-routing-backends>=7.2.0,<8.0.0'
```

Comment:
Aspects also sets tries to install another version of `edx-event-routing-backends` [here](https://github.com/openedx/tutor-contrib-aspects/blob/17966bd8f4119ff0287c3372698caf950ef58416/tutoraspects/patches/openedx-dockerfile-post-python-requirements#L4-L5).

We should probably clarify which version we want to install.

Reply:
Huh, good catch. This was added back in 2022 https://github.com/overhangio/openedx-release-demo/pull/13 and has seen some frequent version upgrades. For the sake of consistency, we can make it the same as aspects i.e. edx-event-routing-backends>=9.3.0,<9.4". 
@ziafazal Hi, do you know if there can be any issues if we update edx-event-routing-backends here?

Reply:
@DawoudSheraz We should remove **event-routing-backends** from `OPENEDX_EXTRA_PIP_REQUIREMENTS` now instead we should rely on aspects to test if **event-routing-backends** is working as expected.

Reply:
we use to test **event-routing-backends** by adding it `OPENEDX_EXTRA_PIP_REQUIREMENTS` when Aspects was not built but now it should be tested with Aspects since Aspects is the only consumer of **event-routing-backends** in open edx ecosystem

Reply:
Alright, thanks

Reply:
I have removed the installation of edx-event-routing-backends from the script.